android/src/main/java/io/ably/flutter/plugin/push/ManualFlutterApplicationRunner.java

```diff
@@ -21,20 +20,27 @@
 import io.flutter.plugin.common.MethodChannel;
 import io.flutter.plugin.common.StandardMethodCodec;
 
-public class PushBackgroundIsolateRunner implements MethodChannel.MethodCallHandler {
-  private static final String TAG = PushBackgroundIsolateRunner.class.getName();
-  private static final String SHARED_PREFERENCES_KEY = "io.ably.flutter.plugin.push.PushBackgroundIsolate.SHARED_PREFERENCES_KEY";
-  private static final String BACKGROUND_MESSAGE_HANDLE_KEY = "BACKGROUND_MESSAGE_HANDLE_KEY";
+/**
+ * This class is used when the application was terminated when the push notification is received.
+ * See [PushMessagingEventHandlers.java] to see where push notifications being handled whilst the
+ * app is in the background or the foreground.
+ *
+ * This class can generified to launch the app manually for any purpose, but currently it
+ * launches the users app and sends it a RemoteMessage.
```

Comment:
"launches the app in use"

Reply:
c2384f5cef7ff1b04033d599516e0b293a32151b